doctr/io/elements.py

```diff
@@ -253,6 +256,79 @@ def synthesize(self, **kwargs) -> np.ndarray:
 
         return synthesize_page(self.export(), **kwargs)
 
+    def export_as_xml(self, file_title: str = 'docTR - XML export (hOCR)', **kwargs) \
```

Comment:
Either we should remove those `**kwargs` since they're not used for now, or pass it down to something

Reply:
Also I realized that this won't work with rotated bbox

Reply:
We should add a `ValueError` when someone attempts the XML export with rotated bbox for now

Reply:
@fg-mindee 
do we have any information about this in `elements.py` ?

Reply:
you can check whether those are rotated bbox dynamically by checking the format of the geometry actually

Reply:
@fg-mindee 
Do you have an example ?
Is with rotated mean vertical and horizontal boxes ?